pkg/ocm/ocm_client.go

```diff
@@ -0,0 +1,87 @@
+package ocm
+
+import (
+	"encoding/json"
+	"os"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/openshift-online/ocm-sdk-go/logging"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	hivev1 "github.com/openshift/hive/apis/hive/v1"
+)
+
+type Client interface {
+	GetAWSAccountClaim(clusterID string) (*awsv1alpha1.AccountClaim, error)
+	GetClusterDeployment(clusterID string) (*hivev1.ClusterDeployment, error)
+}
+
+type client struct {
+	Conn   *sdk.Connection
+	logger *sdk.GlogLogger
+}
+
+func NewOcmClient(ocmConfigFile string) (Client, error) {
```

Comment:
You can simplify a but here so the if statement is shorter

Reply:
a what?

Reply:
I meant, change the if statements to use os.LookupEnv() which returns a boolean
and split the declaration of the envvar to the if statement